src/displayapp/screens/WatchFaceInfineat.cpp

```diff
@@ -434,12 +434,13 @@ void WatchFaceInfineat::Refresh() {
 
   batteryPercentRemaining = batteryController.PercentRemaining();
   isCharging = batteryController.IsCharging();
-  if (batteryController.IsCharging()) { // Charging battery animation
-    chargingBatteryPercent += 1;
+  if (batteryController.IsCharging() && (xTaskGetTickCount() - chargingAnimationTick > pdMS_TO_TICKS(150))) { // Charging battery animation
+    chargingBatteryPercent += 100 / lv_obj_get_height(logoPine);
```

Comment:
I think a code comment here to explain what is going on would be good (dividing 100 by the height means we can find the battery percentage change needed to shift the indicator by 1 pixel)

Reply:
Hopefully done 😀

Reply:
Maybe pop the comment the line before rather than inline? Might be easier to read that way

Reply:
Thank you for your patience. I have been battling the clang formatting to get the test to pass. I always wanted to move the comment above the line of code.